src/main/java/core/basesyntax/HelloWorld.java

```diff
@@ -4,5 +4,12 @@
  * Feel free to remove this class and create your own.
  */
 public class HelloWorld {
-
+    public static void main(String[] args) {
+        FigureSupplier fs = new FigureSupplier();
```

Comment:
```suggestion
        FigureSupplier figureSupplier = new FigureSupplier();
```

fs - not informative variable name